src/App.js

```diff
@@ -1,5 +1,123 @@
+import * as MissionUtils from "@woowacourse/mission-utils";
```

Comment:
MissionUtils에서 Console과 Random과 사용하기에 해당 기능들만 import하는 것이 어떨까 의견남겨봅니다!
```suggestion
import { Console, Random } from "@woowacourse/mission-utils";
```

Reply:
두 방식에 어떤 차이가 있는지 정확하게 잘 몰라서, import export 관련해서도 추가로 공부가 필요할 것 같습니다! 감사합니다

Reply:
사실 저도 처음에는 위의 방식으로 접근했었습니다! 개인적으로 가장 크게 느낀 장점은

기존의 방식은 MissionUtils.Random ~, MissionUtils.Console로 접근했다면

`import { Console, Random } from "@woowacourse/mission-utils";` 의 경우
Console~ , Random~ 이렇게 접근할 수 있어 간결하다고 느꼈습니다.